fritz

```diff
@@ -110,7 +110,7 @@ def check_config(cfg="fritz.defaults.yaml", yes=False, **kwargs):
             config["kowalski"]["server"]["JWT_EXP_DELTA_SECONDS"]
         ),
     )
-    config["skyportal"]["app"]["kowalski"]["token"] = kowalski_token
+    config["skyportal"]["app"]["kowalski"]["token"] = kowalski_token.decode("utf-8")
```

Comment:
Ah, I've finally realized what's going on: please revert this change and instead bump the required version for `pyjwt>=2.0` in fritz/.requirements/dev.txt. You probably have 1.7.x installed locally -- they made this change in 2.0. The nice thing is that the CI actually caught this!

Reply:
Yup I just came to the same conclusion. Reverted and requirements have been bumped